.github/workflows/ci.yaml

```diff
@@ -113,7 +103,11 @@ jobs:
         env:
           KUBECONFIG: ./newrelic-infra-operator/kubeconfig
         run: |
-          helm install newrelic-infra-operator ./helm-charts-newrelic/charts/newrelic-infra-operator --values ./newrelic-infra-operator/values-dev.yaml --set image.repository=localhost:5000/newrelic-infra-operator
+          helm repo add newrelic https://helm-charts.newrelic.com
+          helm repo update
+          helm install newrelic-infra-operator newrelic/newrelic-infra-operator \
```

Comment:
Not sure if it's better to pull from the repo. The old way allows to pull from custom/PR branch to run some tests.

Reply:
I reverted the change, I thought it was there simply because we forgot about bumping it after we merged the branch to master. But it is true it is faster and allows to change the branch